Mediapipe.Net/Framework/Format/ImageFrame.cs

```diff
@@ -49,6 +49,19 @@ unsafe public ImageFrame(ImageFormat format, int width, int height, int widthSte
             Ptr = ptr;
         }
 
+        public ImageFrame(ImageFormat format, int width, int height, int widthStep, ReadOnlySpan<byte> pixelData)
+        {
+            fixed (byte* pixelDataPtr = pixelData)
+            {
+                UnsafeNativeMethods.mp_ImageFrame__ui_i_i_i_Pui8_PF(
+                    format, width, height, widthStep,
+                    pixelDataPtr,
+                    releasePixelData,
+                    out var ptr).Assert();
+                Ptr = ptr;
```

Comment:
This is indeed exactly how it's supposed to work, but do you have any solution to remove the code duplication in the 2 constructors? I mean yeah it's just a function followed by an assignment but still.

Reply:
None that I am aware of, sadly, but we'll cross that bridge when we get to it.

Reply:
Done via https://github.com/vignetteapp/MediaPipe.NET/pull/20/commits/39dfc6cc5b8a5c722432b8218fe9836893edd494